.ddev/.disabled-services/docker-compose.chromedriver.yaml

```diff
@@ -6,13 +6,24 @@
 version: '3.6'
 services:
   chromedriver:
-    image:  previousnext/chrome-headless:65
+    image: seleniarm/standalone-chromium:4.1.4-20220429
+    shm_size: 2gb
     container_name: ddev-${DDEV_SITENAME}-chromedriver
+    expose:
+      - 7900
+    ports:
+      - 5900:5900
```

Comment:
I was sure ddev had dynamic port exports, but it's not here or in our database container. Perhaps port conflicts are only a problem for people who commonly run multiple sites concurrently.